lib/kennel.rb

```diff
@@ -127,10 +132,10 @@ def generated
           load_all
 
           projects = Models::Project.recursive_subclasses.map(&:new)
-          filter_resources!(projects, :kennel_id, project_filter, "projects", "PROJECT")
+          Kennel::Filter.filter_resources!(projects, :kennel_id, filter.project_filter, "projects", "PROJECT")
```

Comment:
might make sense to:
```suggestion
          filter.filter_projects!(projects)
```
so the filter can hold have all the "PROJECT" logic

Reply:
working on that ...

Reply:
https://github.com/grosser/kennel/pull/192